package/mainflux/users.conf

```diff
@@ -1,2 +1,4 @@
 MF_USERS_LOG_LEVEL=info
 MF_USERS_DB_USER=postgres
+MF_USERS_ADMIN_EMAIL=mainflux@mainflux.com
+MF_USERS_ADMIN_PASSWORD=mainflux
```

Comment:
LGTM, but I would keep the blank line at the end of this file.

Reply:
done